src/mjolnir/adminbuilder.cc

```diff
@@ -75,83 +49,47 @@ int polygondata_comparearea(const void* vp1, const void* vp2) {
   return 1;
 }
 
-std::vector<std::string> GetWkts(std::unique_ptr<Geometry>& mline) {
-  std::vector<std::string> wkts;
-
-#if 3 == GEOS_VERSION_MAJOR && 6 <= GEOS_VERSION_MINOR
-  auto gf = GeometryFactory::create();
-#else
-  std::unique_ptr<GeometryFactory> gf(new GeometryFactory());
-#endif
-
-  LineMerger merger;
-  merger.add(mline.get());
-#if GEOS_VERSION_MAJOR == 3 && GEOS_VERSION_MINOR >= 9
-  std::vector<std::unique_ptr<LineString>> merged(merger.getMergedLineStrings());
-#else
-  std::unique_ptr<std::vector<LineString*>> merged(merger.getMergedLineStrings());
-#endif
-  WKTWriter writer;
-
-  // Procces ways into lines or simple polygon list
-#if GEOS_VERSION_MAJOR == 3 && GEOS_VERSION_MINOR >= 9
-  polygondata* polys = new polygondata[merged.size()];
-#else
-  polygondata* polys = new polygondata[merged->size()];
-#endif
+std::string GetWkt(std::vector<polygon_t>& polygons) {
+  std::ostringstream wkt;
+
+  // something to hold the polygons for processing
+  polygondata* polys = new polygondata[polygons.size()];
```

Comment:
should change that to `polygondata polys[polygons.size()]`. already deleted the `delete` further down or this `new`